localstack-core/localstack/testing/pytest/stepfunctions/fixtures.py

```diff
@@ -22,6 +22,52 @@ def sfn_snapshot(snapshot):
     return snapshot
 
 
+@pytest.fixture
+def sfn_batch_snapshot(sfn_snapshot):
```

Comment:
Are we collecting all sfn fixtures here for consistency or is there another reason why we should ship this with the community image, assuming it's only used in ext tests?

Reply:
We collect all sfn fixtures here only for consistency